hieradata/node/bastion1.cp.lsst.org.yaml

```diff
@@ -1,3 +1,3 @@
 ---
 classes:
-  - "x2go"
+  - "profile::core::x2go_agent"
```

Comment:
This probably isn't needed assuming a node has a proper role assigned to it. This is a case where what's in the tree is an anti-example.  The `x2go` should never have been included directly at the node level and instead brought in via a role.